entrypoint.sh

```diff
@@ -5,5 +5,8 @@ set -o errexit
 # pmm-managed-init validates environment variables.
 pmm-managed-init
 
+/usr/share/percona-dashboards/import-dashboards.py -d&
```

Comment:
I'm not sure that it's a good idea to run whole this script each time.

Reply:
Only one function is executed due to used argument "-d".